### PR TITLE
group policy settings  is available in win 10 pro

### PR DESCRIPTION
**Pull request by RAJU2529**
I just added extra line,  because i have applied group policy settings  in windows 10 pro and enterprise.
Thanking you

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix  Copy editing is needed for this PR. Thanks.

File: windows/client-management/mandatory-user-profile.md [hunk 1]
```diff
@@ -150,6 +150,8 @@ When a user is configured with a mandatory profile, Windows 10 starts as though
 | Computer Configuration > Administrative Templates > Windows Components > Search > **Allow Cortana** = Disabled | ![supported](images/checkmark.png) | ![supported](images/checkmark.png) | ![not supported](images/crossmark.png)  | ![not supported](images/crossmark.png)  |
 | Computer Configuration > Administrative Templates > Windows Components > Cloud Content > **Turn off Microsoft consumer experience** = Enabled | ![supported](images/checkmark.png) | ![not supported](images/crossmark.png) | ![not supported](images/crossmark.png) | ![not supported](images/crossmark.png) |
 
+Note Above Group Policy settings can be Applied in windows 10 professional edition
```
Comment (nenonix) [inline windows/client-management/mandatory-user-profile.md]:
```suggestion
> [!Note]
>The Group Policy settings above can be applied in Windows 10 Professional edition.
```

File: windows/client-management/mandatory-user-profile.md [hunk 2]
```diff
@@ -150,6 +150,8 @@ When a user is configured with a mandatory profile, Windows 10 starts as though
...
+
```
Comment (nenonix) [inline windows/client-management/mandatory-user-profile.md]:
```suggestion
```

**Comment by e0i** [comment]
@RAJU2529 Please follow-up with suggested copy editing changes. Thanks.

**Comment by illfated** [comment]
Looks like this is ready for final review (maybe author & manager Daniel Simpson can verify this).

**Comment by e0i** [comment]
@dansimp A community contributed PR is available for review. Thanks.

**Comment by e0i** [comment]
@Dansimp Any updates with the PR? Thanks.

**Comment by e0i** [comment]
@Dansimp A community contributed PR is pending for review. Thanks.

**Pull request merged**